tests/scripts/setup-pytest-env.sh

```diff
@@ -29,9 +29,24 @@ set -u
 export TVM_PATH=`pwd`
 export PYTHONPATH="${TVM_PATH}/python"
 
-export TVM_PYTEST_RESULT_DIR="${TVM_PATH}/build/pytest-results"
+BUILD_DIR="${TVM_PATH}/build"
+export TVM_PYTEST_RESULT_DIR="${BUILD_DIR}/pytest-results"
+rm -f "${BUILD_DIR}"/failed_tests
 mkdir -p "${TVM_PYTEST_RESULT_DIR}"
 
+# This ensures that all pytest invocations that are run through run_pytest will
+# complete and errors will be reported once Bash is done executing all scripts.
+function cleanup() {
+    if [ -f "${BUILD_DIR}"/failed_tests ]; then
```

Comment:
could we track whether this script created that failed_tests file and only error if so?